package/MDAnalysis/core/AtomGroup.py

```diff
@@ -4497,7 +4497,7 @@ def load_new(self, filename, **kwargs):
 
         if not reader:
             # Check if we need to use Chain reader
-            if util.iterable(filename):
+            if not isinstance(filename, util.NamedStream) and util.iterable(filename):
```

Comment:
It could, but a stream is also iterable in a way, it's just not iterable when we think of it as a filename.  I'll have a look at refactoring this a little